doc/decisions/STEPS.md

```diff
@@ -0,0 +1,105 @@
+# Steps
+
+This document describes all steps a decision can run through.
+
+```mermaid
+flowchart LR
+  s((Start)) --> Drafts --> In_Discussion --> In_Progress -- merge --> Decided
+    -- merge --> Partially_Implemented --> Implemented
+
+  %% Shortcuts:
+  s --> Decided
+  Drafts -- merge --> Decided
+  In_Discussion -- merge --> Decided
+  Decided -- merge --> Implemented
+```
```

Comment:
This needs a bit more explanation. What does the "merge" on the edge mean?

I guess, what you want to say is:

1. The first PR for a decision can be created as "Drafts", "In Discussion", "In Progress" or "Decided"
2. A decision PR that wants to merge a decision as "Decided" must:
    1. Already start with the decision as "Decided"
    2. Already have all the considered alternatives fully explored
    3. Only contain discussions/reviews about deciding between one of the considered alternatives
    4. If there is any significant change in the PR (i.e. information changes, not just typos) then the PR cannot be merged as "Decided". The decision must be moved back to "In Progress"
3. After a decision is merged as "Decided", it can be moved to "Partially Implemented" or "Implemented" by any PR.

If that is correct, maybe add this text or something similar in addition to the diagram.

Reply:
Thanks for the summary, I added it in bbe44da7c9d821ece2aecc2a1c400061302cf9d6

Furthermore I wrote:
> The label `merge` on the edges mean that a decision PR must be merged before it can target the next step.